src/appleseed/foundation/math/sampling/qmcsamplingcontext.h

```diff
@@ -256,7 +293,8 @@ inline void QMCSamplingContext<RNG>::split_in_place(
     const size_t        dimension,
     const size_t        sample_count)
 {
-    assert(m_sample_count == 0 || m_instance == m_sample_count);    // can't split in the middle of a sequence
+    // can't split in the middle of a sequence
```

Comment:
Since this is now a full line comment, it must start with a capital and end with a dot.